resources/views/profile/view.blade.php

```diff
@@ -0,0 +1,46 @@
+@extends('layouts.main_layout')
+
+@section('head')
+    <title>{{$username}}s Profile</title>
+@endsection
+
+@section('mainBody')
+    @include('inserts.breadcrumb')
+    <div class="container-fluid well">
+        <legend>Profile</legend>
+        <div class="row">
+		    <div class="col-md-2">
+		        <img class="center-block" scr="images/projects/DefaultProfile.jpg">
+		    	<div class="">
+					<h2 class="center"> {{$username}}</h2>
+					<div class="btn-group btn-group-justified">
+			        	<a href="#" class="btn btn-default">Send Message</a>
```

Comment:
You should have a check here to see if the user is logged in and if they are the owner of this project. If they are you should hide this button.